engine/XcodeUnitTestEngine.php

```diff
@@ -50,7 +51,9 @@ private function shouldGenerateCoverage() {
     //        available.
     // true:  User passed --coverage.
     // https://secure.phabricator.com/T10561
-    return $this->getEnableCoverage() !== false;
+    $arcCoverageFlag = $this->getEnableCoverage();
```

Comment:
Keep the existing function, comment, and logic about parsing the `--coverage` flag from arcanist.